### PR TITLE
[pluto] - setpoint styling improvements

### PR DESCRIPTION
**Pull request by emilbon99**
# Issue Pull Request

## Key Information

- **Linear Issue**: [SY-1427](https://linear.app/synnax/issue/SY-1427/change-setpoint-units-in-schematic)

## Description

Fixes the schematic setpoint overflow in the schematic, improves the styling, correct issues with defining units.

## Basic Readiness

- [x] I have performed a self-review of my code.
- [x] I have added relevant tests to cover the changes to CI.
- [x] I have added needed QA steps to the [release candidate](/synnaxlabs/synnax/blob/main/.github/PULL_REQUEST_TEMPLATE/rc.md) template that cover these changes.
- [x] I have updated in-code documentation to reflect the changes.
- [x] I have updated user-facing documentation to reflect the changes.

### Data Structures

I have ensured that previous versions of stored data structures are properly migrated to new formats in the following projects:

- [ ] Server
- [ ] Console

### API Changes

The following projects have backwards-compatible APIs:

- [ ] Python Client
- [ ] Server
- [ ] TypeScript Client

### CODE REVIEW

File: pluto/src/vis/schematic/primitives/Primitives.css
Comment (pjdotson) [inline pluto/src/vis/schematic/primitives/Primitives.css:188]: Uncomment or delete

**Review by pjdotson — CHANGES_REQUESTED** [review]
image

- Handle on the right is placed below the rest of the setpoint, making it hard to see.
- Units overflow does not work well
- Remove inner orientation selector from style form
- Remove state channel and the state channel logic from setpoint
- Add a size selector to the actual setpoint itself, so it can be more than one size.

**Comment by emilbon99** [comment]
DONE - Handle on the right is placed below the rest of the setpoint, making it hard to see.
DONE - Units overflow does not work well
Remove inner orientation selector from style form
Remove state channel and the state channel logic from setpoint
Add a size selector to the actual setpoint itself, so it can be more than one size.

File: pluto/src/vis/schematic/registry.tsx
Comment (pjdotson) [inline pluto/src/vis/schematic/registry.tsx:1]: Why does this file exist?

**Review by pjdotson — CHANGES_REQUESTED** [review]
Units overflow still looks really bad
image

**Comment by emilbon99** [comment]
Screenshot 2024-12-25 at 7 41 58 PM

@pjdotson Did you build pluto? Set point looks fine to me

File: pluto/src/vis/schematic/registry.ts
Comment (pjdotson) [inline pluto/src/vis/schematic/registry.ts:457]: You have to make sure you don't pass `clickable` to `Primitives.ReliefValve`, otherwise an error for an unknown DOM property gets logged in the console. This should be fixed for all of the other ones too.

**Review by pjdotson — APPROVED** [review]
Approve after merge conflict and last comment is addressed

**Pull request merged**
